src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -787,7 +1079,7 @@ public WorkManager getWorkManager()
      *
      * @param event {@link TickEvent.WorldTickEvent}
      */
-    public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
+    void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "onWorldTick" is 17 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)